src/questions/intro-to-lending-borrowing.yaml

```diff
@@ -1,19 +1,402 @@
-- uuid: 3c35c4d1-4b6d-4071-9019-070426f5bf42
+- uuid: 68d9898f-4838-4218-9356-dba391a63fdd
   type: SingleChoice
-  content: What is the main purpose of DeFi?
-  hint: Purpose of DeFi in the current scenario
-  explanation: Decentralised Finance is responsible for providing financial solutions for existing problems in the centralised system using blockchain.
+  content: What is Collateral?
+  hint: NoHint
+  explanation: Collateral is something pledged as security for repayment of a loan, to be forfeited in the event of a default.
+  answerKeys:
+    - C
+  subTopics:
+    - why-do-we-lending-borrowing
+  difficultyLevel: Low
+  choices:
+    - content: Collateral is net returns on lent assets.
+      key: A
+    - content: Collateral refers to the net worth of the loan.
+      key: B
+    - content: Collateral is something pledged as security for repayment of a loan, to be forfeited in the event of a default.
+      key: C
+    - content: None of these
+      key: D
+
+- uuid: f46a773b-9198-4e13-bc5d-9d9f5ef6d7d5
+  type: SingleChoice
+  content: Why is there the need for over-collateralization?
+  hint: NoHint
+  explanation: Over-collateralization is when you put up more collateral than what is needed to cover a loan. This extra collateral can help to cover potential losses in case of default. Having over-collateralization can help to reduce the risk of default and ensures the security of assets.
+  answerKeys:
+    - C
+  subTopics:
+    - why-do-we-lending-borrowing
+  difficultyLevel: Low
+  choices:
+    - content: Over-collateralization helps to find bugs in the smart-contracts.
+      key: A
+    - content: Over-collateralization helps to increase the interest rates for lenders.
+      key: B
+    - content: Over-collateralization is needed to cover potential losses in case of default. The extra collateral helps to cover the loan in case the loan value exceeds the value of collateral.
+      key: C
+    - content: None of these
+      key: D
+
+- uuid: 9b09ff86-e6d3-4861-987d-aa819fe5622a
+  type: MultipleChoice
+  content: What are the use cases of lending and borrowing?
+  hint: NoHint
+  explanation: Lending increases the value of assets by gaining interest on the lent assets and borrowing helps you to get money in urgent situations or for improvement purposes when capital is insufficient. Borrowing increases an investment's potential return through leveraging.
+  answerKeys:
+    - A
+    - C
+    - D
+  subTopics:
+    - why-do-we-lending-borrowing
+  difficultyLevel: Low
+  choices:
+    - content: Lending increases the value of assets by gaining interest.
+      key: A
+    - content: Lending improves decentralization and security.
+      key: B
+    - content: Borrowing helps you to get money in urgent situations or for improvement purposes when capital is insufficient.
+      key: C
+    - content: Borrowing increases an investment's potential return through leveraging.
```

Comment:
What do you mean by this line `Borrowing increases an investment's potential return through leveraging.` ?  we should use simple english

Reply:
Also is this a correct option?